James pre-release readthrough (#37)
apeltzer/eager2-paper: content/05.methods.md

The pipeline can be started using (raw) FASTQ files from sequencing or
pre-mapped BAM files. Additionally, the pipeline requires a FASTA reference
genome. If BAM input is provided, an optional conversion to FASTQ is offered,
otherwise BAM files processing will start from the post-mapping stage.

If users have complex set-ups, e.g. multiple sequencing lanes that require

> merging of files, the pipeline can be supplied with a tabu separated value (TSV)

Reviewer: ```suggestion
merging of files, the pipeline can be supplied with a tab-separated values (TSV)
```

file to enable such complex data handling. Both FASTQs and BAMs can be provided
in this set up. FASTQs with the same library name and sequencing chemistry but
sequenced across multiple lanes will be concatenated after AdapterRemoval and
prior mapping. Libraries with the sample name and with the same UDG treatment,
will be merged after deduplication. If libraries with the sample name have
different UDG treatment, these will be merged after the aDNA modification stage

> (i.e. BAM trimming or PMDtools, if turned on), prior genotyping, as shown in in

Reviewer: ```suggestion
(i.e. BAM trimming or PMDtools, if turned on), prior genotyping, as shown in 
```

Figure {@fig:merging-files}.

![Schematic of different processing and merging points based on the nature of
different libraries, as specified in metadata of a TSV file. Dashed boxes
represent optional library-specific
processes](images/Fig3_merging_files.png){#fig:merging-files width="70%"}

As Nextflow will automatically download files from URLs, profiles and/or TSV
files can include links to publicly available data (e.g. the ENA FTP server
[@url:ftp://ftp.ebi.ac.uk/pub/databases/ena/]). This assists in reproducibility

> as if profiles or TSV files are uploaded with a publication, a researcher

Reviewer: I don't like this `as if` bit. Consider changing to `since if`?

Reviewer: or `because if`

wishing to re-analyse the data in the same way can use the exact settings and

> merging procedures in the original publication, without having to reconstruct

Reviewer: ```suggestion
merging procedures as in the original publication, without having to reconstruct
```

this from prose.

#### Monitoring

Users can either monitor their pipeline execution with real-time messages

> Nextflow prints to the console while running, or utilize projects such as

Reviewer: ```suggestion
Nextflow prints to the console while running, or utilise projects such as
```
Haven't these been corrected already?
